source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm

```diff
@@ -54,7 +54,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
         dataSource = [[ACRChoiceSetViewDataSourceCompactStyle alloc] initWithInputChoiceSet:choiceSet rootView:rootView];
         [choiceSetView setSeparatorStyle:UITableViewCellSeparatorStyleNone];
     } else {
-        dataSource = [[ACRChoiceSetViewDataSource alloc] initWithInputChoiceSet:choiceSet];
+        dataSource = [[ACRChoiceSetViewDataSource alloc] initWithInputChoiceSet:choiceSet WithHostConfig:config];
```

Comment:
>ceSet:choiceSet WithHostConfig:co [](start = 74, length = 33)

Can we add unit tests for this API/call at all ? or is that not written to support unit tests at this point? If not can we add the sample card + host config you were working with to the samples folder if it doesnt already exist? thanks! Basically whatever it is we need to do to ensure this is part of (eventual automated) test suites/plan that we have.